portfolio/src/main/webapp/style.css

```diff
@@ -1,23 +1,48 @@
+/* Define color scheme of page */
 :root {
     --primary: #ffcece;
     --primary-dark: #ffc1c8;
     --accent: #ffe3b0;
     --contrast: #8ed6ff;  
 }
-
+/* BACKGROUND IMAGES STYLES */
+#body {
+    background-position: center;
+    background-image: url(/images/background/sunset3.jpg);
+    background-attachment: fixed;
+    min-height: 100%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 100vh
```

Comment:
Use a semicolon after every declaration.

It is not strictly required for CSS syntax, but we should follow the style guide in general.

See https://google.github.io/styleguide/htmlcssguide.html#Declaration_Stops

Same for several other styles below